Gemfile

```diff
@@ -30,6 +30,12 @@ gem "rake", "10.1.1", :require => false
 # Hiera has an unbound dependency on json_pure
 # json_pure 2.0.2+ officially requires Ruby >= 2.0, but should have specified that in 2.0
 gem 'json_pure', '~> 1.8', :require => false
+# i18n support (gettext-setup and dependencies)
+gem 'text', '~> 1.3.1', :require => false
+gem 'locale', '~> 2.1.2-2', :require => false
+gem 'gettext', '~> 3.2.2', :require => false
+gem 'fast_gettext', '~> 1.1.0', :require => false
+gem 'gettext-setup', '~> 0.6', :require => false
```

Comment:
Do we need to list all of the dependencies? I'm used to only listing the gems that puppet directly requires & calls, but not the transitive dependencies.